test/cql-pytest/test_compaction_tombstone_gc.py

```diff
@@ -0,0 +1,141 @@
+# Copyright 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+from util import new_materialized_view, new_test_table
+import nodetool
+import time
+
+def test_tombstone_gc_with_conflict_in_memtable(scylla_only, cql, test_keyspace):
+    """
+    Regression test for fixed https://github.com/scylladb/scylladb/issues/20423
+    """
+    schema = "k int, v int, primary key (k, v)"
+    with new_test_table(cql, test_keyspace, schema, extra="with gc_grace_seconds = 0") as table:
+        with nodetool.no_autocompaction_context(cql, table):
+            nodetool.setlogginglevel(cql, "compaction", "trace")
```

Comment:
Please remove this setlogginglevel from the test or comment it out (you can leave the function definition in nodetool.py for future use, if you want). This isn't needed for this test - just perhaps helped you debug its failures - and after setting this trace value does not undo it and will leave it for the rest of the test suite, causing a lot of crap in the log.

Reply:
right, will do